tests/acceptance/stepDefinitions/userContext.js

```diff
@@ -0,0 +1,44 @@
+const {When, Then} = require('@cucumber/cucumber')
+const {client} = require('nightwatch-api')
+const axios = require("axios");
+
+When('the user navigates to the homepage using the webUI', function () {
+    return client.url(client.launchUrl)
+});
+
+Then('the user should see sign up button', function () {
```

Comment:
```suggestion
Then('the signup button should be visible on the webUI', function () {
```
Maybe something like this?